src/test/jtx/impl/Env.cpp

```diff
@@ -189,6 +192,9 @@ Env::AppBundle::~AppBundle()
     app->getJobQueue().rendezvous();
     app->signalStop();
     thread.join();
+
+    // Remove the debugLogSink before the suite goes out of scope.
+    setDebugLogSink (nullptr);
```

Comment:
I'm gathering from the discussion between @ximinez and @vinniefalco, the best implementation   for this may be outside of `Env`.  I'm fine with that, if that's what you agree to.  Or we could throw the change out, since it only provides a minor improvement in code coverage.